app/livechat/server/api/v1/videoCall.js

```diff
@@ -33,9 +32,6 @@ API.v1.addRoute('livechat/video.call/:token', {
 				throw new Meteor.Error('invalid-livechat-config');
 			}
 
-			Messages.createWithTypeRoomIdMessageAndUser('livechat_video_call', room._id, '', guest, {
```

Comment:
If you enable video call in Livechat, Jitsi configuration changes, as it does not have the option to open the jitsi call in the new tab. That means as soon as the agent clicks on the phone icon, the jitsi call starts at the same window(side space), so in this case the agent does not need an action-link(join call button) to join the call, the agent is already in call in the current window. But I talked with murtaza and he told me we need to have a join call button there. So, I need to make changes in code. But I think the error is in this function Messages.createWithTypeRoomIdMessageAndUser only because as I remove this function api is working fine. I am able to get responses as shown in the above image at widget side.                                                                                                                   

No, I am not allowing visitors to initiate the call. Above response I got from api after the visitor clicks on the accept button at widget side and response will be used for generating a link for jitsi at widget side.